src/lib/desktop.ts

```diff
@@ -466,10 +466,13 @@ export async function writeProjectSettingsFile(
 }
 
 // Important for saving settings.
-// TODO: should be pulled from electron-builder.yml
```

Comment:
I don't think this value is changing enough to justify the complexity of reading from a file. It's also hard-coded in a few other scripts. If we ever changed this value it would be an easy, global find-and-replace.